server/core/admin.cc

```diff
@@ -271,6 +272,35 @@ std::string get_filename(const HttpRequest& request)
     return path;
 }
 
+static const char *get_ssl_version(const enum admin_ssl_version_t ssl_version)
+{
+    const char *ver;
+
+    if (ssl_version == ADMIN_SSL_VERSION_NORMAL)
```

Comment:
Use a `switch` statement and handle all the known cases separately. This will prevent this code from compiling when new enumeration values are added.